test/nodes/test_dataset_cobalt.py

```
@@ -0,0 +1,588 @@
+from types import ModuleType
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+
+# ---------------------------------------------------------------------------
+# Mock infrastructure: rocketlib, engLib, ai.common, cobalt, depends
+# ---------------------------------------------------------------------------
+
+
+def _install_mocks():
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_install_mocks`

Add return type annotation: `None`

(ANN202)